modules/phenyl-api-explorer/src/client/config/env.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-env node */
```

Comment:
What's the main reason you didn't use flow or ES module import / export here?

Reply:
This file is copy of create-react-app.